mycroft/skills/core.py

```diff
@@ -459,6 +459,40 @@ def show_url(self, url, override_idle=None):
         self["url"] = url
         self.show_page("SYSTEM_UrlFrame.qml", override_idle)
 
+    def show_config(self, setting_key, setting_type, setting_value,
```

Comment:
Should this be called `register_setting` maybe since it doesn't affect the screen, but instead registers a setting for the skill config page.